client/src/components/App.js

```diff
@@ -1,14 +1,13 @@
-import React, { Component } from "react";
-import NavBar from "./NavBar";
-import Temperature from "./Temperature";
-import Login from "./Login";
-import "bootstrap/dist/css/bootstrap.min.css";
-import "../css/App.css";
-import { appConfig } from "../utils/constants";
-import { UserSession } from "blockstack";
-import Button from "react-bootstrap/Button";
-import { configure, User, getConfig } from "radiks";
-import Patient from "../models/patient";
+import React, { Component } from 'react';
+import Login from './Login';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '../css/App.css';
+import { appConfig } from '../utils/constants'
+import { UserSession } from 'blockstack';
+
```

Comment:
might want to remove this new line here for code cleanliness but it's fine regardless 👍